01/12/24 (12:18:33) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:18:34) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:18:42) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:18:48) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:18:53) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:18:58) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:19:04) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:19:10) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
01/12/24 (12:19:18) | ERROR: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
             ^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/baolhq/.config/bpytop/themes'
